concepticondata/conceptlists/Yao-2017-1100.tsv

```diff
@@ -0,0 +1,1101 @@
+Yao-2017-1100-719	719	人质	hostage			2.64	5.72	5.31	5.67	5.53	5.72
+Yao-2017-1100-720	720	仁爱	benevolence			7.53	5.93	6.67	6.63	6.87	6.55
+Yao-2017-1100-721	721	仁慈	benevolent			6.29	5.71	6.61	6.5	7.77	7.01
+Yao-2017-1100-722	722	任务	mission			4.38	4.29	4.19	4.76	4.76	5.47
+Yao-2017-1100-723	723	日常	daily life			2.63	5.9	4.29	5.03	4.62	4.76
+Yao-2017-1100-724	724	日历	calendar	CALENDAR	1967	5.29	3.54	4.57	6.28	5	5.36
+Yao-2017-1100-725	725	日期	date			4.18	3.55	5.55	5.18	6.11	5.26
+Yao-2017-1100-726	726	荣幸	honor	HONOR (SOMEBODY)	3628	2.38	7.18	5.68	5.18	6.76	6.31
+Yao-2017-1100-727	727	容貌	appearance	APPEARANCE	1017	6.15	5.27	5.83	5.51	6.02	5.71
+Yao-2017-1100-728	728	容易	easy	EASY	686	6	6	6.81	5.83	6.74	6.12
+Yao-2017-1100-729	729	榕树	banyan tree			5.41	5.95	6.55	6.5	4.66	4.79
+Yao-2017-1100-730	730	融合	combine			4.95	3.68	6.85	7.59	8.31	6.97
+Yao-2017-1100-731	731	冗长	wordy			3.93	4.76	6.02	6.44	6.51	5.97
+Yao-2017-1100-732	732	柔情	tenderness			6.54	5.27	6.34	6.02	7.31	6.52
+Yao-2017-1100-733	733	柔软	soft	SOFT	1856	7.47	6.53	4.94	4.31	5.7	5.65
+Yao-2017-1100-734	734	柔弱	weak and tender			1.5	6.25	4.14	4	3.95	4.61
+Yao-2017-1100-735	735	肉搏	fist fight			6.61	6.18	4.46	3.32	5.69	6.31
+Yao-2017-1100-736	736	睿智	wise	WISE	698	2.71	6.89	4.46	5.29	4	5.24
+Yao-2017-1100-737	737	撒谎	lie	LIE (MISLEAD)	601	5.07	4.9	3.95	4.09	3.18	3.61
+Yao-2017-1100-738	738	丧服	memorial suit for the dead			1.67	7.33	4.25	4.78	4.67	4.81
+Yao-2017-1100-739	739	丧礼	funeral	FUNERAL	3336	1.93	8.32	5.89	6.39	7.43	6.53
+Yao-2017-1100-740	740	丧失	lose	LOSE	1660	6.73	6.23	6.5	5.17	6.28	5.67
+Yao-2017-1100-741	741	丧亡	pass away	DIE	1494	1.22	7.36	4.42	4.31	5.51	5.28
+Yao-2017-1100-742	742	扫兴	bummer			2.2	5.36	4.73	4.5	5.53	5
+Yao-2017-1100-743	743	色斑	color spot			3.64	5.89	3.83	4.97	5.05	4.79
+Yao-2017-1100-744	744	色情	porn			3.14	7.71	4.79	4.36	4.16	5.16
+Yao-2017-1100-745	745	杀戮	killing			2.61	5.54	4.73	5.41	6.03	6.09
+Yao-2017-1100-746	746	沙尘	sand	SAND	671	4.66	4.61	5.15	5.34	5.82	5.52
+Yao-2017-1100-747	747	沙漠	desert	DESERT	3687	2.3	7.4	6.3	4.43	6.86	6.2
+Yao-2017-1100-748	748	傻瓜	fool			3.22	3.68	5.32	5.51	6.51	5.66
+Yao-2017-1100-749	749	傻子	fool	FOOL	1438	3	2.7	5.57	4.9	6.97	5.8
+Yao-2017-1100-750	750	山崩	landslide			4.82	3.71	3.82	3.68	4.97	4.47
+Yao-2017-1100-751	751	山峰	mountain	MOUNTAIN	639	1.94	8.58	4.39	5.25	4.47	5.11
+Yao-2017-1100-752	752	善良	kind	KIND	3299	7.83	5.53	5.3	6.77	5.46	6.03
+Yao-2017-1100-753	753	伤疤	scar	SCAR	497	6.67	5.14	4.56	4.66	6.2	5.26
+Yao-2017-1100-754	754	伤痕	scar			2.51	6.14	5.28	4.4	5.65	5.46
+Yao-2017-1100-755	755	伤口	scar	WOUND	1109	5.11	5.96	4.82	2.73	4.42	4.47
+Yao-2017-1100-756	756	伤势	condition for the injury			5.27	5.53	5	6.2	5.72	5.71
+Yao-2017-1100-757	757	烧伤	burn	BURN	2102	2.63	7.03	4.83	5.5	4.79	5.23
+Yao-2017-1100-758	758	稍微	a bit	A LITTLE	2924	2.54	5.44	5.37	3.97	6.1	5.25
+Yao-2017-1100-759	759	少女	young lady	GIRL	1646	6.57	5.37	6.93	6.8	6.64	6.17
+Yao-2017-1100-760	760	申请	apply	APPLY (OINTMENT)	3650	5.79	3.18	6.75	6.82	7.1	6.18
```

Comment:
If the dictionaries I found are any good, this seems to be more like 'apply for (e.g. a job)'

Reply:
Yes, this means exactly "sich bewerben" or similar.